### ISSUE TITLE
[FR] Automatically Populate Current Tab Title When Editing Tab Title

### ISSUE THREAD

**Issue by N3C2L** [madprops/grasshopper#94]
Greetings!

I hope you're doing well.

I understand that there's a feature freeze for this repository, and I completely respect that.

This small FR is my only one for the next time, promise. I'd appreciate it if you could spare a moment for it.

When attempting to edit a tab title (for a tab that was loaded for the first time) using the "edit title" command, the tab title field always appears empty (if the tab title hasn't been changed previously).

I'm aware of another command that copies the tab title: copy > copy title.

My small feature request here is, that this actual tab title from the "copy title" will be automatically inserted into the "edit title" prompt, and automatically selected.

This would mirror the behavior of pressing ctrl+d in native Firefox to prompt the bookmark dialogue (where the text for the current tab title is automatically inserted and selected).

Why is that important for me? Often I wouldn't need to retype text from the already pre-inserted tab title.

**Comment by madprops** [comment]
Hi again N3C2L

I have added that functionality as per your request.

The title is auto-filled and selected.

Also added a setting in `More` called `Edit Title Auto` which is enabled by default.

Also added a Fill button to the edit prompt. Which in the case of the title fills it with the real/original title when pressed.

As an added note I also added two commands `Select Up` and `Select Down` which select all items above or below the current item. Added two shortcuts for this ctrl+shift+up and ctrl+shift+down. This can be useful when you filter and want to select all items to close tabs etc. This was already possible with shift+home and shift+end but the commands were missing.
